<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParamDialog</class>
 <widget class="QDialog" name="ParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="paramView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>200</width>
     <height>340</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="reagentBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>320</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>试剂参数</string>
   </property>
   <widget class="QCheckBox" name="enableCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>是否使用</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantityLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加样量</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加样速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="delay1Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>延时</string>
    </property>
   </widget>
   <widget class="QLabel" name="pumpLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>蠕动泵速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="delay2Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>延时</string>
    </property>
   </widget>
   <widget class="InputEdit" name="quantityEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="InputEdit" name="speedEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="InputEdit" name="delay1Edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="InputEdit" name="pumpEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="InputEdit" name="delay2Edit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-10)</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantityLabel2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-10)mL</string>
    </property>
   </widget>
   <widget class="QLabel" name="delay1Label2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-200)秒</string>
    </property>
   </widget>
   <widget class="QLabel" name="delay2Label2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-200)秒</string>
    </property>
   </widget>
   <widget class="QLabel" name="pumpLabel2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="sampleBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>320</width>
     <height>290</height>
    </rect>
   </property>
   <property name="title">
    <string>样本过滤</string>
   </property>
   <widget class="QLabel" name="pumpSpeedLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>蠕动泵速度</string>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>延时</string>
    </property>
   </widget>
   <widget class="InputEdit" name="pumpSpeedEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="InputEdit" name="delayEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-200)秒</string>
    </property>
   </widget>
   <widget class="QLabel" name="pumpSpeedLabel2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(0-10)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="ImageLabel" name="newLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>178</width>
     <height>58</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageLabel" name="openLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>185</y>
     <width>178</width>
     <height>58</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageLabel" name="saveLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>178</width>
     <height>58</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageLabel" name="saveAsLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>315</y>
     <width>178</width>
     <height>58</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageLabel" name="cancelLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>140</width>
     <height>56</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>140</width>
     <height>56</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>component/imagelabel.h</header>
   <slots>
    <signal>released()</signal>
    <signal>pressed()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>InputEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">component/inputedit.h</header>
   <slots>
    <signal>released()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
